### PR TITLE
add close on fileread finish to tracee-rules

### PR DESCRIPTION
**Pull request by AsafEitani**

### CODE REVIEW

**Comment by AsafEitani** [comment]
Fixed issue of tracee-rules exiting before signatures finished printing.

File: tracee-rules/engine/engine.go [hunk 1]
```diff
@@ -219,7 +234,7 @@ func (engine *Engine) LoadSignature(signature types.Signature) (string, error) {
 		engine.logger.Printf("error initializing signature %s: %v", metadata.Name, err)
 
 	}
-	go signatureStart(signature, c)
+	go signatureStart(signature, c, &engine.waitGroup)
```
Comment (yanivagman) [inline tracee-rules/engine/engine.go:237]: Shouldn't you add `engine.waitGroup.Add(1)` before this line?

Comment (itaysk) [reply tracee-rules/engine/engine.go:237]: preferably inside signature start

Comment (AsafEitani) [reply tracee-rules/engine/engine.go:237]: I am :)
Thanks

**Review by yanivagman — APPROVED** [review]

**Pull request merged**